### ISSUE TITLE
Support for ES Modules

### ISSUE THREAD

**Issue by thebuilder** [JeringTech/Javascript.NodeJS#151]
Hi, and thanks for a really awesome library - Really helps bridge the gap between Node and .NET.

Is it possible to add support for ES Modules/`.mjs` files? 

Currently, trying to execute code in `.mjs` files fail, with:

```
Error [ERR_REQUIRE_ESM]: require() of ES Module .../server/entry-server.mjs not supported.
```

It's executed using:

```
await _nodeJsService.InvokeFromFileAsync<string>("../server/entry-server.mjs", "renderHead");
```

The whole Node system is slowly moving toward ES modules as the standard. In our case we're using Vite to bundle our server code, and it will by default output an `.mjs` file.

**Comment by JeremyTCD** [comment]
### TL;DR

The crashing was occurring for `InvokeFromFileAsync` calls.  

After some investigation, it turns out the crashing is *not* an issue for end-users like yourself!  

### Details
Some background on how this library works - it starts a HTTP server in Node.js ([server logic](https://github.com/JeringTech/Javascript.NodeJS/blob/8f3a07fb9f744347996b6ea763d183cdb93c2b69/src/NodeJS/Javascript/Servers/OutOfProcess/Http/Http11Server.ts)). It uses HTTP requests to send JS invocations to the Node.js process. When you call `InvokeFromFileAsync`,  it tells the Node.js server to use a dynamic import to load your file ([line here](https://github.com/JeringTech/Javascript.NodeJS/blob/8f3a07fb9f744347996b6ea763d183cdb93c2b69/src/NodeJS/Javascript/Servers/OutOfProcess/Http/Http11Server.ts#L122)).   

The crashing only occurs when the server logic is web-packed in development mode. In that mode, webpack uses eval to execute the server logic. Turns out there is a [Node.js issue of segmentation faults](https://github.com/nodejs/node/issues/43205) when using dynamic imports in evaluated code.

When the server logic is web-packed in production mode, eval is not used and so the crashing does not occur. For end-users consuming this library as a Nuget package, the crashing is not an issue because the server logic is web-packed in production mode for the Nuget package.
